views/pulsar/v2/helpers/nav.html.twig

```diff
@@ -56,12 +56,12 @@
 {% endmacro %}
 
 
-{% macro nav_item(options) %}
+{% macro nav_item(options, index) %}
 {% spaceless %}
     {% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
     {% import '@pulsar/pulsar/v2/helpers/util.html.twig' as util %}
 
-    <li
+    <li id="nav-item-{{ index }}"
```

Comment:
Could we get caught out here if there are multiple instances of the nav? (not sure if that is a possibility), example - 2 nav items with the id `nav-item-0`. Do we need to namespace instances of the primary nav?